lua/lush/importer.lua

```diff
@@ -1,17 +1,26 @@
+local group_pattern = "[@.%w_]+"
```

Comment:
`.` should be `%.` for the literal dot?

Reply:
My understanding is that you don't need to escape inside character groups. From empirical testing it doesn't seem to match arbitrary strings which would happen if it needed to be escaped.

Reply:
Yep seems correct!